custom/icds/management/commands/migrate_pse_attendance.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
+
+from django.core.management.base import BaseCommand
+
+from django.db import connections
+
+
+new_table = 'config_report_icds-cas_static-child_cases_monthly_v2_198ccc06'
+old_table = 'config_report_icds-cas_static-child_cases_monthly_tabl_551fd064'
+
+migration_query = """
+UPDATE "{new_table}" B
+SET
+pse_daily_attendance = A.pse_days_attended,
+pse_daily_attendance_male = CASE WHEN sex = 'M' THEN A.pse_days_attended ELSE NULL END,
+pse_daily_attendance_female = CASE WHEN sex = 'F' THEN A.pse_days_attended ELSE NULL END
+FROM "{old_table}" A
+WHERE A.doc_id = B.doc_id and A.month = immutable_date_cast(B.month_start)
```

Comment:
is there any other way to do this join, on indexed columns? if we don't do it on indexed columns it's probably going to take a very long time